File: abc/abc397/a/main.js
const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.question('', (input) => {
  const X = parseFloat(input);

  if (X >= 38.0) {
    console.log(1);
  } else if (X >= 37.5) {
    console.log(2);
  } else {
    console.log(3);
  }

  rl.close();
});

File: abc/abc397/a/main.rs
use std::io;

fn main() {
  let mut input = String::new();
  io::stdin().read_line(&mut input).expect("入力エラー");
  let x: f64 = input.trim().parse().expect("数値として解析できません");

  if x >= 38.0 {
    println!("1");
  } else if x >= 37.5 {
    println!("2");
  } else {
    println!("3");
  }
}